### PR TITLE
How to test Angor

### PR DESCRIPTION
**Pull request by Danielogenna**
Added how to create a project if you are a founder.

### CODE REVIEW

File: src/content/posts/How-to-test-Angor/index.md [hunk 1]
```diff
@@ -17,88 +17,107 @@ Angor is a decentralized crowdfunding project leveraging the security of Bitcoin
...
-## Step 3: Testing Milestone Verification and Fund Release
+### Step 3: Creating and Managing a Project (For Founders)
 
-### 1. Monitor Project Progress
-**Steps:**
+#### Create a New Project
+Steps:
+- Navigate to the "Create Project" section on Angor.
+- Enter the project name, description, and goals.
+- Define the project milestones and the corresponding time-lock contracts.
```
Comment (dangershony) [inline src/content/posts/How-to-test-Angor/index.md:58]: Maybe also mention to add a banner this is important for projects view page

File: src/content/posts/How-to-test-Angor/index.md [hunk 2]
```diff
@@ -17,88 +17,107 @@ Angor is a decentralized crowdfunding project leveraging the security of Bitcoin
...
+- Click "Submit" to create the project.
+
+#### Manage Project Milestones
+Steps:
+- Regularly update the project’s progress on Angor.
```
Comment (dangershony) [inline src/content/posts/How-to-test-Angor/index.md:63]: the projects updates go on nostr not angor, we still dont have a way to post updates form angor by the founder can take the private key and import it to a nostr client and there post updates.

**Review by dangershony — APPROVED** [review]

File: src/content/posts/How-to-test-Angor/index.md (hunk 1 above)
Comment (Danielogenna) [reply src/content/posts/How-to-test-Angor/index.md:58]: Yes will commit that

File: src/content/posts/How-to-test-Angor/index.md (hunk 2 above)
Comment (Danielogenna) [reply src/content/posts/How-to-test-Angor/index.md:63]: Will also adjust to this.

**Pull request closed without merging**